src/util/cliUtil.js

```diff
@@ -0,0 +1,28 @@
+const findUp = require('find-up');
+const fs = require('fs-extra-promise');
+const path = require('path');
+
+const SITE_CONFIG_NAME = 'site.json';
+
+module.exports = {
+  findRootFolder: (userSpecifiedRoot, siteConfigPath = SITE_CONFIG_NAME) => {
+    if (userSpecifiedRoot) {
+      const resolvedUserSpecifiedRoot = path.resolve(userSpecifiedRoot);
+      const expectedConfigPath = path.join(resolvedUserSpecifiedRoot, siteConfigPath);
+      if (!fs.existsSync(expectedConfigPath)) {
+        throw new Error(`Config file not found at user specified root ${resolvedUserSpecifiedRoot}`);
+      }
+      return resolvedUserSpecifiedRoot;
+    }
+
+    const currentWorkingDir = process.cwd();
+    // Enforces findUp uses value of process.cwd() to determine starting dir
+    // This allows us to define starting dir when testing by mocking process.cwd()
+    const foundConfigPath = findUp.sync(siteConfigPath, { cwd: currentWorkingDir });
+    if (!foundConfigPath) {
+      throw new Error(`No config file found in parent directories of ${currentWorkingDir}`);
+    }
+    const foundRootPath = path.dirname(foundConfigPath);
```

Comment:
Seems like this const just for readability. 

Since it is implied that the site.config is found successfully already, we can just return

`return path.dirname(foundConfigPath)`